releases/release/list.go

```diff
@@ -47,6 +48,23 @@ func (d *Requirements) Validate() error {
 	return nil
 }
 
+func (d *Requirements) QueryParams() (params map[string]string) {
```

Comment:
This only uses the struct tags to avoid the risk of these differing from what is being bound by Gin here - https://github.com/circleci/ex/blob/2f11180309bb7319d64499c20d88455f14516c65/releases/release/handler.go#L81

Maybe this is overkill ?